Sources/Fullscreen/FavoriteAdsList/FavoriteAdsListView.swift

```diff
@@ -156,8 +197,14 @@ extension FavoriteAdsListView: UITableViewDelegate {
 
     public func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
         tableHeaderView.endEditing(true)
-        tableView.deselectRow(at: indexPath, animated: true)
-        delegate?.favoriteAdsListView(self, didSelectItemAt: indexPath)
+        if !tableView.isEditing {
+            tableView.deselectRow(at: indexPath, animated: true)
+        }
+        selectRow(at: indexPath)
```

Comment:
I think we can just call `delegate?.favoriteAdsListView(self, didSelectItemAt: indexPath)` since it's the only thing we have in `selectRow` method.

Reply:
Yeah, no point in over-engineering this😂 I'll fix it